arch/openrisc/lib/memset.S

```diff
@@ -0,0 +1,95 @@
+/*
+ * OpenRISC memset.S
+ *
+ * Hand-optimized assembler version of memset for OpenRISC.
+ * Algorithm inspired by several other arch-specific memset routines
+ * in the kernel tree
+ *
+ * Copyright (C) 2015 Olof Kindgren <olof.kindgren@gmail.com>
+ *
+ *      This program is free software; you can redistribute it and/or
+ *      modify it under the terms of the GNU General Public License
+ *      as published by the Free Software Foundation; either version
+ *      2 of the License, or (at your option) any later version.
+ */
+
+	.global memset
+	.type	memset, @function
+memset:
+	#Args r3 = *s, r4 = c, r5 = n
```

Comment:
I think we use c-style comment style /**/ in asm, but I've seen # used by other arch's, so that's probably fine.
But please add a space after the #, that makes the comments a lot more pleasant to read.